src/lib/components/parameters.svelte

```diff
@@ -113,6 +120,7 @@
     });
 
     readAllParameters();
+
```

Comment:
One thing on the to-do list (#4) is add some automated formatting checks which should give faster feedback on this sort of thing. In the meantime:

```suggestion
```

Reply:
yay a hint which formater and style is wanted in the repo will sute help. I manual formarted to make it like the rest ;)